src/main/java/sg/ncl/MainController.java

```diff
@@ -307,7 +318,8 @@ public void subscriptionAgreementDownload(HttpServletResponse response) throws M
     @RequestMapping(value = "/UsagePolicy/download", method = RequestMethod.GET)
     public void usagePolicyDownload(HttpServletResponse response) throws UsagePolicyDownloadException, IOException {
         InputStream stream = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)